packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -45,6 +45,7 @@
   "Accounts_Default_User_Preferences_audioNotifications": "Audio Notifications Default Alert",
   "Accounts_Default_User_Preferences_desktopNotifications": "Desktop Notifications Default Alert",
   "Accounts_Default_User_Preferences_mobileNotifications": "Mobile Notifications Default Alert",
+  "Accounts_Default_User_Preferences_not_available": "Failed to retrieve User Preferences because of they haven't been set up by the user yet",
```

Comment:
One tiny thing on message would be: 
> because of they haven't been set up by the user yet
To 
> because they haven't been set up by the user yet

Alternative to set up might "initialized" I think doesn't matter though.

Reply:
@geekgonecrazy yes, my bad there. I don't know where that `of` came from really!!! Anyways fixed now.